tensorflow-examples/src/main/java/org/tensorflow/model/examples/tensors/TensorCreation.java

```diff
@@ -0,0 +1,62 @@
+package org.tensorflow.model.examples.tensors;
```

Comment:
Missing license/copyright... and that make me realize that it was also missing in most of the files in this repo, which I just updated now. You can take the same one as in [this file](https://github.com/tensorflow/java-models/blob/master/tensorflow-examples/src/main/java/org/tensorflow/model/examples/mnist/SimpleMnist.java).

Reply:
Agree, will do